framework/core/instance.cpp

```diff
@@ -22,11 +22,23 @@
 #include <algorithm>
 #include <functional>
 
+#if defined(VKB_DEBUG) || defined(VKB_VALIDATION_LAYERS)
+#	define USE_VALIDATION_LAYERS 1
+#endif
+
+#if defined(USE_VALIDATION_LAYERS) && (defined(VKB_VALIDATION_LAYERS_GPU_ASSISTED) || defined(VKB_VALIDATION_LAYERS_BEST_PRACTICES) || defined(VKB_VALIDATION_LAYERS_SYNCHRONIZATION))
+#	define USE_VALIDATION_LAYER_FEATURES 1
+#endif
+
+#ifdef USE_VALIDATION_LAYERS
+#	define USE_VULKAN_LOGGER
+#endif
```

Comment:
Do we want to centralize this logic instead of copying it to two places?

Reply:
I am reluctant to add more headers that would couple the CPP and C frameworks together. Later I imagine we will chose one framework over the other. These macros are less likely to change than the duplicated code in both frameworks and so there is relatively less risk keeping them duplicated as there is making a framework change.

For now I would like to keep them copied

Reply:
fair enough on both accounts.